File: mobile/src/main/java/net/stecky/puzzlemethis/events/BehaviorSubType.java
package net.stecky.puzzlemethis.events;

/**
 * Created by sb185219 on 11/9/2017.
 */

public class BehaviorSubType extends EventSubType
{
    private boolean isGood;

    public BehaviorSubType(String name, int icon, boolean isGood)
    {
        super(name, icon);
        this.isGood = isGood;
    }

    public boolean isGood()
    {
        return isGood;
    }

    public void setGood(boolean good)
    {
        isGood = good;
    }
}

File: mobile/src/main/java/net/stecky/puzzlemethis/events/behaviors/BehaviorSubType.kt
package net.stecky.puzzlemethis.events.behaviors

import android.widget.ImageView
import net.stecky.puzzlemethis.events.EventSubType

/**
 * Created by sb185219 on 11/14/2017.
 */
class BehaviorSubType(name: String, icon: ImageView, var good: Boolean) : EventSubType(name, icon)
{
}